kliff/calculators/calculator_torch.py

```diff
@@ -121,18 +122,40 @@ def create(
                 nprocs,
             )
 
+        # Finally, assign fingerprints dataset property as a FingerprintsDataset instance
+        self.fingerprints_dataset = FingerprintsDataset(self.fingerprints_path)
+
+    def get_fingerprints_dataset(self):
+        """
+        Return a list of fingerprints of the configurations.
+        """
+        return self.fingerprints_dataset.fp
```

Comment:
The function name and the return are inconsistent. Can we rename the function to `get_fingerprints`?